python/ray/serve/_private/logging_utils.py

```diff
@@ -228,8 +231,10 @@ def record_factory(*args, **kwargs):
         logs_dir = get_serve_logs_dir()
     os.makedirs(logs_dir, exist_ok=True)
 
-    max_bytes = ray._private.worker._global_node.max_bytes
-    backup_count = ray._private.worker._global_node.backup_count
+    if not max_bytes:
+        max_bytes = ray._private.worker._global_node.max_bytes
+    if not backup_count:
```

Comment:
```suggestion
    if backup_count is None:
```
Have to be careful -- `backup_count=0` is valid